evm_arithmetization/src/proof.rs

```diff
@@ -968,59 +968,45 @@ impl BurnAddrTarget {
         }
     }
 
+    #[cfg(feature = "cdk_erigon")]
```

Comment:
This would require to conditionally remove this field from `PublicValuesTarget` but I think we'd prefer keep it consistent with the non target version